crates/matrix-sdk-ui/src/room_list_service/mod.rs

```diff
@@ -516,7 +516,7 @@ fn configure_all_or_visible_rooms_list(
     list_builder
         .sort(vec!["by_recency".to_owned(), "by_name".to_owned()])
         .filters(Some(assign!(SyncRequestListFilters::default(), {
-            is_invite: Some(with_invites),
+            is_invite: if with_invites { None } else { Some(false) },
```

Comment:
I think this warrants a comment (bonus points if you link to the spec, which I had to read to understand the difference): `Some(false)` means only non-invites, while not defined means both invites and non-invites. Can you add that please?